src/modules/timepix3/geometry/Config.cpp

```diff
@@ -53,6 +53,11 @@ Config::Config(std::string ConfigFile) {
   } catch (...) {
     LOG(INIT, Sev::Warning, "Using default MaxTimeGapNS");
   }
+  try {
+    FrequencyHz = root["FrequencyHz"].get<uint16_t>();
```

Comment:
float?

Reply:
I'm looking for frequency in Hz, it should be an integer

Reply:
I don't understand what you mean: 14.54 Hz is also a frequency but not  an integer.

Reply:
I cannot argue with that, change will come soon